shared/css/user-menu.scss

```diff
@@ -193,4 +200,16 @@
       padding-left: 10px;
     }
   }
+
+  @media screen and (max-width: 1023px) {
```

Comment:
This is controlling the Create/New Project menu button.

Reply:
Thanks for labelling these changes!